crowdfunding_compassion/templates/project_donation_page.xml

```diff
@@ -22,8 +22,8 @@
                                         <div class="card h-100 card-body d-flex justify-content-around align-items-center">
                                             <span class="donation-type__card-icon"/>
                                             <h4 class="blue text-center uppercase">
-                                                Create impact by
-                                                <t t-esc="project.product_id.crowdfunding_impact_text_active"/>
+                                                Create impact by supporting
+                                                <t t-esc="project.product_id.name"/>
```

Comment:
No, please restore the previous version. You were not seeing the text because you didn't configure your product correctly and set the `crowdfunding_impact_text_active`